File: app/src/main/java/com/quibbler/sevenmusic/listener/BroadcastDatabaseListener.java
package com.quibbler.sevenmusic.listener;

/**
 * Package:        com.quibbler.sevenmusic.listener
 * ClassName:      BroadcastDatabaseListener
 * Description:
 * Author:         11103905
 * CreateDate:     2019/9/27 18:54
 */
public interface BroadcastDatabaseListener extends BroadcastListener {
    public void onDatabaseChanged();
}

File: app/src/main/java/com/quibbler/sevenmusic/listener/BroadcastDatabaseListener.kt
package com.quibbler.sevenmusic.listener

/**
 * Package:        com.quibbler.sevenmusic.listener
 * ClassName:      BroadcastDatabaseListener
 * Description:
 * Author:         11103905
 * CreateDate:     2019/9/27 18:54
 */
interface BroadcastDatabaseListener : BroadcastListener {
    fun onDatabaseChanged()
}
